### PR TITLE
Added contact form section

### PR DESCRIPTION
**Pull request by Aquinasmoriasi**
- Added contact form section
- Added name input field that accepts a maximum of 30 characters
- Added an email input field
- Added a messaging field that accepts no more than 500 characters
- Added the server where the input data is sent to
- Included a submission button
- Added background styling to the form and also styled the whole section
- Added client-side form validation
- Added button effects i.e, hover, disabled and active

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -293,8 +293,38 @@ <h4><a href="#">LET'S CONNECT </a></h4>
...
+            <h5>Contact me</h5>
+            <p>
+            If you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding.
+            I’d love to help with it
+            </p>
+            <form class="input" action="https://formspree.io/f/xlezvgbv" method="post">
+               <input type="text" class="input-item" name="Name" placeholder="full name" maxlength="30">
+               <input type="email" class="input-item" name="Email" placeholder="email">
+               <textarea name="message" cols="30" rows="10" placeholder="write your message here" maxlength="500"></textarea>
```
Comment (Whoistolu) [inline index.html:308]: - [ ] [REQUIRED] Kindly ensure all input fields have the required attribute to activate the validation on submit. This is a core requirement for this project.

File: styles.css [hunk 2]
```diff
@@ -618,3 +618,125 @@ body {
...
+  height: 3rem;
+  border-style: none;
+  border-radius: 0.5rem;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 24px;
+  color: #6070ff;
+  margin-bottom: 2.8125rem;
+  padding-bottom: 0;
+}
```
Comment (Whoistolu) [inline styles.css:696]: - [ ] [REQUIRED] Let's ensure that the button interactions such as hover, enabled, disabled, and enabled, pressed are styled as shown in the Figma template1.
### An Iage of Template 1
Screenshot from 2022-06-08 12-31-34

File: index.html [hunk 3]
```diff
@@ -293,8 +293,38 @@ <h4><a href="#">LET'S CONNECT </a></h4>
...
+    <!-- This is the footer section of the webpage-->
+    <footer>
+        <section id="contact-form">
+            <h5>Contact me</h5>
+            <p>
+            If you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding.
+            I’d love to help with it
+            </p>
+            <form class="input" action="https://formspree.io/f/xlezvgbv" method="post">
```
Comment (Whoistolu) [inline index.html:305]: - [ ] [OPTIONAL] Kindly ensure you follow the contact form styling properly. There is a horizontal overflow currently and a background image on the text area. Let's fix this for a better user experience UX.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:

Screenshot from 2022-06-08 12-34-12 | Screenshot from 2022-06-08 12-38-00

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Aquinasmoriasi 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Aquinasmoriasi, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Aquinasmoriasi** [comment]
Thank you

**Pull request merged**